tools/cloud-build/provision/trigger-schedule/README.md

```diff
@@ -0,0 +1,35 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
Same here, would it make sense for us to have a little more description of what this does?